packages/rich-text/src/component/index.js

```diff
@@ -235,7 +235,7 @@ class RichText extends Component {
 		if ( onPaste ) {
 			// Only process file if no HTML is present.
 			// Note: a pasted file may have the URL as plain text.
-			const image = find( [ ...items, ...files ], ( { type } ) =>
+			const image = filter( [ ...items, ...files ], ( { type } ) =>
```

Comment:
Filter implies that an array would be returned, right? So this needs to be `images`?